lib/exsoda/http.ex

```diff
@@ -199,10 +199,48 @@ defmodule Exsoda.Http do
         headers(op),
         http_options
       )
-      |> as_json
+      |> maybe_202(path, op, fn -> post(path, op, body) end)
     end
   end
 
+  defp poll202(path, op, ticket, redo) do
+    :timer.sleep(10000) # should this be configurable?
```

Comment:
10 seconds?  Doesn't seem super-long to me, for jobs that core will toss to freezer.